config.js

```diff
@@ -60,6 +60,19 @@ const config = {
             doneColumnId: 15345341,
           },
         },
+        {
```

Comment:
Maybe you can get rid of the 1.7.8.1 then?

Reply:
I approve @atomiix

Reply:
done !